libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -465,8 +474,8 @@ bool ASTJsonConverter::visit(ElementaryTypeName const& _node)
 bool ASTJsonConverter::visit(UserDefinedTypeName const& _node)
 {
 	setJsonNode(_node, "UserDefinedTypeName", {
-		make_pair("name", namePathToString(_node.namePath())),
-		make_pair("referencedDeclaration", idOrNull(_node.annotation().referencedDeclaration)),
+		make_pair("pathNode", toJson(*_node.pathNode())),
+		make_pair("referencedDeclaration", idOrNull(_node.pathNode()->annotation().referencedDeclaration)),
```

Comment:
We could remove it, but maybe it is better to keep it to make this "less breaking".